ios/FluentUI.Demo/FluentUI.Demo/Demos/ListItemDemoController_SwiftUI.swift

```diff
@@ -202,23 +209,22 @@ struct ListItemDemoView: View {
                 if renderStandalone {
                     listItem
                 }
-                List {
+                FluentList {
                     if !renderStandalone {
-                        Section {
+                        FluentListSection {
                             listItem
                         } header: {
                             Text("ListItem")
                         }
                     }
                     controls
                 }
-                .background(ListItem.listBackgroundColor(for: .grouped))
+                .fluentListStyle(listStyle)
                 .fluentTheme(fluentTheme)
-                .listStyle(.insetGrouped)
             }
         }
 
-        return content
+        return content.background(ListItem.listBackgroundColor(for: .grouped))
```

Comment:
why'd we have to move the background here?

Reply:
This was a part of validation and should not be moved. Reverted.